modules/ROOT/pages/lb-ssl-endpoints.adoc

```diff
@@ -0,0 +1,102 @@
+= Configure SSL Endpoints and Certificates
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: cloudhub, runtime manager, arm, load balancing, ssl, two way tls,
+
+A dedicated load balancer must have at least one certificate associated with it. To configure an SSL endpoint for your load balancer to serve to clients, provide a certificate and private key pair for your load balancer to serve to clients. 
```

Comment:
Does this sentence need "for your load balancer to serve to clients"? Trying to find a way to not repeat "your load balancer to serve to clients."